### PR TITLE
div. Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/views/templates/copyElements/_copyPackagesAndIEs.gsp [hunk 1]
```diff
@@ -132,7 +132,7 @@
                             <div data-pkgoid="${genericOIDService.getOID(sp)}">
                                 <div class="ui checkbox la-toggle-radio la-replace">
                                     <g:checkBox name="subscription.takePackageIds"
-                                                value="${genericOIDService.getOID(sp)}" data-pkgid="${sp.id}"
+                                                value="${genericOIDService.getOID(sp)}" data-pkgoid="${sp.id}"
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/copyElements/_copyPackagesAndIEs.gsp:135]: pkgoid and pkg.id?

File: app/grails-app/views/templates/copyElements/_copyPackagesAndIEs.gsp [hunk 2]
```diff
@@ -121,7 +121,7 @@
 
                                         <div class="ui checkbox la-toggle-radio la-replace">
                                             <g:checkBox name="subscription.takePackageSettings"
-                                                        value="${genericOIDService.getOID(sp)}" data-pkgid="${sp.id}"
+                                                        value="${genericOIDService.getOID(sp)}" data-pkgoid="${sp.id}"
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/copyElements/_copyPackagesAndIEs.gsp:124]: pkgoid and pkg.id?

File: app/grails-app/views/templates/cpa/_personFormModal.gsp [hunk 3]
```diff
@@ -11,201 +12,208 @@
...
                                       value="${org?.id}"
                                       optionKey="id"
                                       optionValue=""/>
                         </g:if>
-                        <g:if test="${org}">
-                            <label for="personRoleOrg">
-                                <g:message code="person.belongs.to"/>
+                        <g:else>
+                            <label for="positionOrg">
+                                <g:message code="contact.belongesTo.label"/>
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/cpa/_personFormModal.gsp:96]: belongsTo

**Review by croqueGrec09 — COMMENTED** [review]
questions

**Review by croqueGrec09 — APPROVED** [review]
approved

**Review by croqueGrec09 — APPROVED** [review]
approved

File: app/grails-app/views/templates/subscription/_subscriptionTable.gsp [hunk 4]
```diff
@@ -232,32 +232,37 @@
...
-                                    </g:link>
+                                        </g:link>
+                                    </g:if>
+                                </g:if>
+                                <g:if test="${contextService.org.getCustomerType()  == 'ORG_CONSORTIUM'}">
+                                    <g:set var="surveysConsortiaSub" value="${SurveyConfig.findBySubscriptionAndSubSurveyUseForTransfer(s ,true)}" />
+                                    <g:if test="${surveysConsortiaSub}">
+                                        <g:link controller="subscription" action="surveysConsortia" id="${s.id}"
+                                                class="ui icon button">
+                                            <g:if test="${surveysConsortiaSub?.surveyInfo?.isCompletedforOwner()}">
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/subscription/_subscriptionTable.gsp:252]: surveysConsortiaSub?

**Review by croqueGrec09 — COMMENTED** [review]
one ? too much

**Review by klodav — APPROVED** [review]

**Pull request merged**